main.tex

```diff
@@ -467,7 +513,9 @@ \section{Results: Numerical experiments}\label{sec:numerical-experiments}
     \caption{\(\SSe\) for all strategies considered.
         A similar conclusion to that of
         Figure~\ref{fig:sserror_in_stewart_plotkin} can be made: the strategies
-        that score highly have a negatively skewed \(\SSe\).}
+        that score highly have a negatively skewed \(\SSe\). The authors
```

Comment:
Can we increase the resolution somehow? I feel like we shouldn't say "this isn't a high quality figure in print".

Reply:
I believe (but please correct me) that the problem isn't resolution but just real estate of a printed page and needing/trying to include more than 200 names of strategies.

> I feel like we shouldn't say "this isn't a high quality figure in print".

I know exactly what you mean. It felt a wrong to write that but at the same time I felt it was not necessarily a bad thing to point out to readers/editors that it's a pity publications are somewhat still constrained by print.

*But* I've got a better suggestion (I believe):

The main thing we want from this figure is looking at the general trend of the scatter plot: we don't actually care about the particular strategies (just good to know that high rank by score have a high low skew etc). So what about this:

- In the paper, let us remove the names of the strategies all together so we just plot skew of SSE vs rank. Without needing to include the names we can have a larger axis label etc...
- We include as auxiliary material (so it would be available online) a pdf of the plot that includes the name: we could then make this as big as we want.
- Change the current caption to be something like: 
    ```
     \caption{\(\SSe\) for all strategies considered plotted against the rank according to score and wins.
    A similar conclusion to that of
    Figure~\ref{fig:sserror_in_stewart_plotkin} can be made: the strategies
    that score highly have a negatively skewed \(\SSe\). A version of this plot including the
    names of all strategies is available in the auxiliary materials.}
    ```

I'm happy to try any further suggestions.

Reply:
That sounds reasonable to me.

Reply:
Cool, I'll do that and put it in the style file. :+1: